velox/docs/functions/spark/datetime.rst

```diff
@@ -29,13 +29,16 @@ These functions support TIMESTAMP and DATE input types.
     If num_days is a negative value then these amount of days will be
     deducted from start_date.
 
-.. spark:function:: datediff(endDate, startDate) -> integer
+.. spark:function:: date_format(timestamp, format) -> string
 
-    Returns the number of days from startDate to endDate. Only DATE type is allowed
-    for input. ::
+    Converts ``timestamp`` to a date/time string (local timezone adjusted) in the format specified by
+    ``format``, such as yyyy-MM-dd HH:mm:ss, yyyy-MM-dd, etc. Invalid ``format`` will cause exception
```

Comment:
What does .etc refer to? Where can the reader find a list of all supported formats?

Reply:
Just revised this document and added a link where valid format pattern is listed.

Reply:
@PHILO-HE Thanks. A link to documented list of formats is nice.